src/status_im2/contexts/chat/messages/pin/events.cljs

```diff
@@ -56,6 +57,24 @@
                  (assoc-in [:pin-message-lists chat-id]
                            (message-list/add-many nil (vals all-messages))))}))))
 
+(rf/defn delete-pinned-message
```

Comment:
I thought there's a `:pin-message/send-pin-message` event to unpin message
example at https://github.com/status-im/status-mobile/blob/637dc4e279da9e8d97fa9a937adce88b0facaf47/src/status_im/ui2/screens/chat/messages/message.cljs#L283

Is this not working?

Last time when I reproduced issue #14729 while working on delete message

I remember the issue is when re-enter the chat view, mobile will load message from status-go and the `:pinned` or another related key turns to `false`
which caused the issue